### PR TITLE
Buffs the Pathfinder Gear

### PR DESCRIPTION
**Pull request by Super3222**
#### Intent of your Pull Request

The pathfinder cloak is currently broken, and doesn't protect it's bearer from ash storms. This PR will fix that.
The cloak will also come with a toggable kasa.
Given the above statement, you can no longer craft kasa's.
The cost for the cloak is now 4 sinew, and 10 blocks of mythril (If feedback is positive,  I'll raise this to 15).

The cost for pathfinder treads has also been reduced. The recipe is now 1 bone, 1 sinew, and 1 block of mythril.

##### Changelog

:cl: Super3222
bugfix: The Pathfinder cloaks will protect you from ashstorms.
tweak: Pathfinder cloaks come hooded with a toggable kasa.
rscdel: You can no longer craft kasa's given that they come with cloaks.
tweak: Pathfinder cloaks are now easier to craft, but more expensive. The requirements are 4 pieces of sinew(from 2), and 10 blocks of mythril. It no longer requires 2 pieces of bone.
tweak: Pathfinder treads are now easier to craft as well. The requirements are 1 piece of bone, 1 piece of sinew (from 3 sinew), and 1 block of mythril.
tweak: Mining Hardsuit is no longer completely burn proof. It's only ash proof.
/:cl:

### CODE REVIEW

File: code/modules/mining/lavaland/weather_types.dm [hunk 1]
```diff
@@ -105,4 +105,6 @@
 		var/thermal_protection = H.get_thermal_protection()
 		if(thermal_protection >= FIRE_IMMUNITY_SUIT_MAX_TEMP_PROTECT)
 			return
+		if((istype(H.wear_suit,/obj/item/clothing/suit/hooded/pathfinder)) && (istype(H.head, /obj/item/clothing/head/helmet/kasa)))
```
Comment (Cruix) [inline code/modules/mining/lavaland/weather_types.dm]: This is way too snowflakey, put a list of weather immunities on all clothing and make it so the pathfinder suit gives ash protection when the hood is up.

Comment (Super3222) [reply code/modules/mining/lavaland/weather_types.dm]: I agree

File: code/__DEFINES/flags.dm [hunk 2]
```diff
@@ -34,6 +34,7 @@
 // BLOCK_GAS_SMOKE_EFFECT only used in masks at the moment.
 #define BLOCK_GAS_SMOKE_EFFECT 8192	// blocks the effect that chemical clouds would have on a mob --glasses, mask and helmets ONLY! (NOTE: flag shared with THICKMATERIAL)
 #define THICKMATERIAL 8192		//prevents syringes, parapens and hypos if the external suit or helmet (if targeting head) has this flag. Example: space suits, biosuit, bombsuits, thick suits that cover your body. (NOTE: flag shared with BLOCK_GAS_SMOKE_EFFECT)
+#define ASH_IMMUNE 20
```
Comment (Cruix) [inline code/__DEFINES/flags.dm]: Why is this 20? That's not a multiple of 2.

File: code/__DEFINES/flags.dm [hunk 3]
```diff
@@ -34,6 +34,7 @@
 // BLOCK_GAS_SMOKE_EFFECT only used in masks at the moment.
 #define BLOCK_GAS_SMOKE_EFFECT 8192	// blocks the effect that chemical clouds would have on a mob --glasses, mask and helmets ONLY! (NOTE: flag shared with THICKMATERIAL)
 #define THICKMATERIAL 8192		//prevents syringes, parapens and hypos if the external suit or helmet (if targeting head) has this flag. Example: space suits, biosuit, bombsuits, thick suits that cover your body. (NOTE: flag shared with BLOCK_GAS_SMOKE_EFFECT)
+#define ASH_IMMUNE 22
```
Comment (Cruix) [inline code/__DEFINES/flags.dm]: That's....still not a power of 2...

Comment (Super3222) [reply code/__DEFINES/flags.dm]: I'm an idiot.

File: code/__DEFINES/flags.dm [hunk 4]
```diff
@@ -34,7 +34,7 @@
 // BLOCK_GAS_SMOKE_EFFECT only used in masks at the moment.
 #define BLOCK_GAS_SMOKE_EFFECT 8192	// blocks the effect that chemical clouds would have on a mob --glasses, mask and helmets ONLY! (NOTE: flag shared with THICKMATERIAL)
 #define THICKMATERIAL 8192		//prevents syringes, parapens and hypos if the external suit or helmet (if targeting head) has this flag. Example: space suits, biosuit, bombsuits, thick suits that cover your body. (NOTE: flag shared with BLOCK_GAS_SMOKE_EFFECT)
-#define ASH_IMMUNE 22
+#define ASH_IMMUNE 65536
```
Comment (Cruix) [inline code/__DEFINES/flags.dm]: This is too big, bitflags in byond can only go up to 32768

**Comment by ShadowDeath6** [comment]
uh
don't ashwalkers have passive immunity to ash storms

**Comment by Super3222** [comment]
Me and Stealth discussed that. Currently the hermit spawns with mythril.

**Comment by Super3222** [comment]
Hm... seems like this PR is actually busted. It's master branch isn't working properly.

**Pull request closed without merging**